acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -107,35 +92,30 @@ def visitHostsPage()
 end
 
 def fillInHostAndPort(host)
-    @hosts_page.hostName_input.set host
+    @hosts_page.hostName_input.set getTableValue(host)
 end
 
 def fillInResourcePool(pool)
-    @hosts_page.resourcePool_input.select pool
+    @hosts_page.resourcePool_input.select getTableValue(pool)
 end
 
 def fillInRAMCommitment(commitment)
-    @hosts_page.ramCommitment_input.set commitment
+    @hosts_page.ramCommitment_input.set getTableValue(commitment)
 end
 
 def clickAddHostButton()
     @hosts_page.addHost_button.click
 end
 
-def removeAllHosts()
-    defaultMatch = Capybara.match
-    Capybara.match = :first
-    while @hosts_page.host_entries.size != 0 do
-        click_link_or_button("Delete")
-        click_link_or_button("Remove Host")
-    end
-    Capybara.match = defaultMatch
-end
-
 def addHost(name, pool, commitment)
     clickAddHostButton()
     fillInHostAndPort(name)
     fillInResourcePool(pool)
     fillInRAMCommitment(commitment)
     click_link_or_button("Add Host")
+end
+
+def addDefaultHost()
+    addHost("table://hosts/defaultHost/nameAndPort", \
+        "table://hosts/defaultHost/pool", "table://hosts/defaultHost/commitment")
```

Comment:
style nit pick - please put "table://hosts/defaultHost/commitment" on a line by itself.  There's many schools of aesthetic-thought here, but the general convention here at Zenoss is to put additional arguments one/line if they don't all fit on the first line